### PR TITLE
Opt 1849

### PR DESCRIPTION
**Pull request by kerilz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +29,15 @@ those variable plugins:
 | [Resolve reference variables](variable-import-plugin/resolve-reference-variables)                      | 2.1.0+               | variable, plugin, import|
 | [Transform complex variables](variable-import-plugin/transform-complex-variables)                      | 2.2.0+               | variable, plugin, import|
 
+### Getting Started with Decision Import Plugins
+
+Analogously to the variable import plugins, you can hook into the decision instandce import to enrich, filter or modify the input and output decision variables. [More information here.](https://docs.camunda.org/optimize/latest/technical-guide/plugins/decision-import/)
```
Comment (megglos) [inline README.md]:
```suggestion
Analogously to the variable import plugins, you can hook into the decision instance import to enrich, filter or modify the input and output decision variables. [More information here.](https://docs.camunda.org/optimize/latest/technical-guide/plugins/decision-import/)
```

File: decision-import-plugin/filter-decision-inputs/README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        boolean validAbilityId = false;
+        try {
+          validAbilityId = isValidAbilityId(i);
+        } catch (UnirestException e) {
+          e.printStackTrace();
+        }
+        return validAbilityId;
+      })
+      .collect(
+        Collectors.toList());
```
Comment (megglos) [inline decision-import-plugin/filter-decision-inputs/README.md:75]: minor: remove line break

File: decision-import-plugin/filter-decision-inputs/README.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  <repository>
+    <id>camunda-bpm-nexus</id>
+    <name>camunda-bpm-nexus</name>
+    <url>
+      https://app.camunda.com/nexus/content/repositories/camunda-optimize
+    </url>
+  </repository>
+</repositories>
+```
+
```
Comment (megglos) [inline decision-import-plugin/filter-decision-inputs/README.md:48]: let's add the third party library note as done in the Readme of the output example

File: decision-import-plugin/resolve-reference-outputs/README.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+Note: Some libraries are already used in Optimize and therefore provided out of the box.
+Instead of including those to an uber jar, just set the scope of those dependencies to
+`provided`. To get an overview, what is already provided by Optimize, have a look at
+the [third party libraries][5] in the Optimize documentation. Using your own version
+of libraries that are already included in Optimize might cause weird side effects otherwise.
+
+### Implement the decision import plugin
+
+Let's say you have a situation where you need to use some of the Pokémon abilities.
+Depending on the ability, you need to decide which Pokémon needs to be summoned.
```
Comment (megglos) [inline decision-import-plugin/resolve-reference-outputs/README.md]: I like you description! Would just mention the term decision definition already here to give context for the next sentence. 
```suggestion
Depending on the ability, a decision definition decides which Pokémon needs to be summoned.
```

File: decision-import-plugin/resolve-reference-outputs/README.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+So, as a desicion output you get an Pokémon ID, which is not really useful for the data analysis, as you want to see
```
Comment (megglos) [inline decision-import-plugin/resolve-reference-outputs/README.md]:
```suggestion
So, as a decision output you get an Pokémon ID, which is not really useful for the data analysis, as you want to see 
```

**Review by megglos — APPROVED** [review]
Nice examples Kyrylo! The Pokemon context makes it fun and outlines the use case of plugins very well!
Noticed only minor things. Preapproving

**Comment by JoHeinem** [comment]
I'm already merging this PR so it can be used in the blog post. Feel free to implement the review hints as a separate commit.

**Pull request merged**